outputs.tf

```diff
@@ -0,0 +1,14 @@
+output "id" {
+  value       = "${aws_cloudtrail.trail.id}"
+  description = "The name of the trail."
+}
+
+output "home_region" {
+  value       = "${aws_cloudtrail.trail.home_region}"
+  description = "The region in which the trail was created."
+}
+
+output "arn" {
+  value       = "${aws_cloudtrail.trail.arn}"
+  description = "The Amazon Resource Name of the trail."
+}
```

Comment:
In the event that we create the S3 bucket, I think that it would be valuable to emit the `id` and `arn` here too. That allows us to fallback on Terraform's default resource dependency resolution (not having to do `depends` everywhere).